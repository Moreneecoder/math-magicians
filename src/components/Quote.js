import '../stylesheets/quote.css';

const Quote = () => (
  <div className="Quote">
    <h1>Quote &apos;Em Up</h1>
    <div className="main main-bg-color">
      “Knowledge is a garden. If it isn’t cultivated, you can’t harvest it.”
    </div>

    <div className="author main-bg-color">
      AFRICAN PROVERB
    </div>
  </div>
);

export default Quote;
