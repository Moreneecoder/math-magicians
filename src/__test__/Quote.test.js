import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Quote Component', () => {
  test('Renders Quote component as expected', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
