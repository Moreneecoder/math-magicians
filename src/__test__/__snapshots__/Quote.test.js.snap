// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component Renders Quote component as expected 1`] = `
<div
  className="Quote"
>
  <h1>
    Quote 'Em Up
  </h1>
  <div
    className="main main-bg-color"
  >
    “Knowledge is a garden. If it isn’t cultivated, you can’t harvest it.”
  </div>
  <div
    className="author main-bg-color"
  >
    AFRICAN PROVERB
  </div>
</div>
`;
